public/locales/en/translation.json

```
@@ -533,6 +550,11 @@
         "FsrSharpnessStrenght": "FSR Sharpness Strength",
         "fsync": "Enable Fsync",
         "gamemode": "Use GameMode (Feral Game Mode needs to be installed)",
+        "gamescope": {
+            "enableLimiter": "Enable FPS Limiter",
+            "enableUpscaling": "Enables Upscaling",
+            "missingMsg": "We could not found gamescope on the PATH. Install it or add it to the PATH."
```

```suggestion
            "missingMsg": "Heroic could not find gamescope on your system. Install it or add it to the PATH."
```

I think it is easier to understand this way.